src/aead/less_safe_key.rs

```diff
@@ -140,20 +175,18 @@ fn open_within_<'in_out>(
     key: &LessSafeKey,
     nonce: Nonce,
     aad: Aad<&[u8]>,
+    received_tag: Tag,
     in_out: &'in_out mut [u8],
     src: RangeFrom<usize>,
+    ciphertext_len: usize,
```

Comment:
You don't need `ciphertext_len` here, as it can be computed as `in_out.get(src).ok_or(error::Unspecified).len()`, right?

Reply:
That works as well, yes, but wasn't this way for the reason I stated above. However, I've made the change to remove the `ciphertext_len` parameter since it probably doesn't matter.